src/Menu/menu.scss

```diff
@@ -0,0 +1,21 @@
+.menu {
+  position: relative;
+}
+.menu-content {
+  position: absolute;
+  top: 100%;
+  z-index: 10;
+  background: #fff;
+  min-width: 10rem;
+  &.pin-left {
+    left: 0;
+  }
+  &.pin-right {
+    right: 0;
+  }
+}
+
+// @import './overlay-menu';
```

Comment:
Leftover?

Reply:
This is unused at the moment, but will likely be used soon